include/IECore/CurvesAlgo.h

```diff
@@ -47,7 +47,7 @@ namespace IECore
 namespace CurvesAlgo
 {
 
-PrimitiveVariable resamplePrimitiveVariable( const CurvesPrimitive &curvesPrimitive, const PrimitiveVariable &primitiveVariable, PrimitiveVariable::Interpolation interpolation );
+void resamplePrimitiveVariable( const CurvesPrimitive *curvesPrimitive, PrimitiveVariable &primitiveVariable, PrimitiveVariable::Interpolation interpolation );
```

Comment:
Seems easily rebasable as well, assuming separate commits for the .h+.cpp and tests